docs/manual-EN/functions-and-operators/functions-and-operator-reference.md

```diff
@@ -1,24 +1,35 @@
-|  ***Name***    |  ***Description***    | 
-|:----|:----:|
-|  abs()      |  Return the absolute value      | 
-|  acos()  |  Return the arc cosine | 
-|  asin()  |  Return the arc sine  | 
-|  atan()  |  Return the arc tangent  | 
-|   cbrt()   |   Returns the cubic root of the argument   | 
-|  ceil()  |  Return the smallest integer value not less than the argument    | 
-|  cos()  |  Return the cosine | 
-|  exp()   |   Raise to the power of   | 
-|  floor()    |  Return the largest integer value not greater than the argument      | 
-|  hypot()   |  Returns the hypotenuse of a right-angled triangle  | 
-|  log()     |   Return the natural logarithm of the first argument    | 
-|  log2()    |  Return the base-2 logarithm of the argument    | 
-|  log10()    |  Return the base-10 logarithm of the argument    | 
-|  now()  |  Return the current date and time  | 
-|  pow()   |   Return the argument raised to the specified power   | 
-|  rand32()  |  Return a random 32 bit interger  | 
-|  rand64()  |  Return a random 64 bit interger  | 
-|  round()  |  Round the argument  | 
-|  sin()  |  Return the sine of the argument  | 
-|  sqrt()     |  Return the square root of the argument    | 
-|  strcasecmp()  | Compare strings without sensitivity to case   | 
-|  tan()  |  Return the tangent of the argument | 
+|  ***Name***        |  ***Description***                                                        |
+|:-------------|:-------------------------------------------------------------------:|
+|  abs         |  Return the absolute value                                          |
+|  acos        |  Return the arc cosine                                              |
+|  asin        |  Return the arc sine                                                |
+|  atan        |  Return the arc tangent                                             |
+|  cbrt        |   Returns the cubic root of the argument                            |
+|  ceil        |  Return the smallest integer value not less than the argument       |
+|  cos         |  Return the cosine                                                  |
+|  exp         |   Raise to the power of                                             |
+|  floor       |  Return the largest integer value not greater than the argument     |
+|  hypot       |  Returns the hypotenuse of a right-angled triangle                  |
+|  length      |  Return the length of a string in bytes                             |
+|  left        |  Return the leftmost number of characters as specified              |
+|  log         |   Return the natural logarithm of the first argument                |
```

Comment:
Why `the first argument`? Is the function supposed to assign multiple values? Please check and unify the expression.